### PR TITLE
[chromecast] Manual Thing Configuration, Port of chromecast audio group is necessary

### PR DESCRIPTION
**Pull request by miwok72**
Clarification on the manual thing configuration, audio groups concerning. I determined the port with "iobroker".

See also: https://community.openhab.org/t/solved-chromecast-audiogroup-unable-to-play-stream-on-group/67059

### CODE REVIEW

File: addons/binding/org.openhab.binding.chromecast/README.md [hunk 1]
```diff
@@ -34,7 +34,13 @@ No authentication is required for accessing the devices on the network.
 ## Thing Configuration
 
 Chromecast devices can also be manually added.
-The only configuration parameter is the `ipAddress`.
+The only configuration parameter is the `ipAddress`. For an audio group also the port is necessary. The autodiscover process finds the port automatically. With manual thing configuration the port must be determined manually.
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.chromecast/README.md]: One typo: "autodiscovery"

```suggestion
The only configuration parameter is the `ipAddress`.
For an audio group also the port is necessary.
The autodiscovery process finds the port automatically.
With manual thing configuration the port must be determined manually.
```

Comment (davidgraeff) [reply addons/binding/org.openhab.binding.chromecast/README.md]: "also the **por" :)

Comment (cweitkamp) [reply addons/binding/org.openhab.binding.chromecast/README.md]: I edited my comment. Thanks.

Comment (cweitkamp) [reply addons/binding/org.openhab.binding.chromecast/README.md]: @miwok72 Did you see my comment? Do you want to apply the suggestion or not?

Comment (miwok72) [reply addons/binding/org.openhab.binding.chromecast/README.md]: Thank you, please apply your suggestion!

Comment (cweitkamp) [reply addons/binding/org.openhab.binding.chromecast/README.md]: @miwok72 May I ask you to do it?

Comment (miwok72) [reply addons/binding/org.openhab.binding.chromecast/README.md]: I'm sorry, that was my first PR and I'm not sure which steps to take

Comment (cweitkamp) [reply addons/binding/org.openhab.binding.chromecast/README.md]: No problem. In https://github.com/openhab/openhab2-addons/pull/4946#discussion_r258961705 you should see a "Commit suggestion" button. Just click on it and the suggestion will be applied.

**Pull request merged**